### PR TITLE
Join Tables

### PR DESCRIPTION
**Pull request by Sanja969**
### Feature 4 - Join Tables

- Created vet table and insert raws with vets' data
- Created join tables for information about visits and vets' specialties and insert data in them
- Answered the following questions:
    Who was the last animal seen by William Tatcher?
    How many different animals did Stephanie Mendez see?
    List all vets and their specialties, including vets with no specialties.
    List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
    What animal has the most visits to vets?
    Who was Maisy Smith's first visit?
    Details for most recent visit: animal information, vet information, and date of visit.
    How many visits were with a vet that did not specialize in that animal's species?
    What specialty should Maisy Smith consider getting? Look for the species she gets the most.

    **Screenshots of the results:**
image
image
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -66,4 +66,82 @@ WHERE owner_id = 5 AND escape_attempts = 0;
...
+WHERE vis.date_of_visit = (select max(date_of_visit) FROM visits WHERE vets_id=1) 
+ORDER BY vis.date_of_visit desc;
+
+SELECT v.name, a.name FROM visits vis
+INNER JOIN animals a
+  ON a.id = vis.animals_id
+INNER JOIN vets v
+  ON v.id = vis.vets_id
+WHERE v.name = 'Stephanie Mendez'
+GROUP BY (v.name, a.name);
```
Comment (AnselemOdims) [inline queries.sql]: - [x] The query says how many different animals did Stephanie Mendez see, but this is only listing them. Please kindly adjust query to match question.

File: queries.sql [hunk 2]
```diff
@@ -66,4 +66,82 @@ WHERE owner_id = 5 AND escape_attempts = 0;
...
+  ON v.id = vis.vets_id
+WHERE v.name = 'Stephanie Mendez' and date_of_visit BETWEEN 'April 1, 2020' and 'August 30, 2020';
+
+SELECT a.name, count(a.name) FROM visits vis
+INNER JOIN animals a
+  ON a.id = vis.animals_id
+INNER JOIN vets v
+  ON v.id = vis.vets_id
+GROUP BY a.name
+order by count desc;
```
Comment (AnselemOdims) [inline queries.sql]: - [x] The query question is what animal has the most visits to vets? Please ensure you only list the animal with the most visits

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Sanja969, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by EricMbouwe — APPROVED** [review]
Hi @Sanja969, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Made requested changes ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**